### PR TITLE
Improve alignment preview image color ranges

### PR DESCRIPTION
**Pull request by psavery**
For datasets that had a minimum greater than zero, the preview images in the alignment widget would often appear saturated. That is because the actual values in the inscribed circle would become much higher than the values outside of it (which we should ignore anyways).

Thus, when we compute the range to include for the color range, only check pixel values inside the inscribed circle (and a certain distance from the edge as well, since edge pixels appear to drop fast in value).

This makes the preview images look much better.

Fixes: #2279

### CODE REVIEW

**Comment by psavery** [comment]
@ercius Unfortunately, I did just notice one other problem too.

If the tilt rotation axis is non-zero, then the image isn't really an inscribed circle anymore, and thus we encounter saturation again:

image

**Comment by ercius** [comment]
Hm. That makes this harder. The wizard is not meant for huge rotation corrections and you probably cant take care of every edge case. I recommend subtracting the minimum before doing this analysis in such a case. So, lets go with your current solution which will be helpful in most experiments.

**Comment by psavery** [comment]
It looks like the images even start appearing more saturated with about a 5 degree rotation.
image

Should I also subtract the minimum from the data before performing the slice reconstruction? Or are we going to leave that for the user to do?

**Comment by ercius** [comment]
I'm wary to subtract the minimum. I sometimes use this to see what the
reconstruction will look like. So, I think we need to encourage users to
set the minimum to zero.

**Comment by psavery** [comment]
Okay! @cjh1 This is ready for your review, then.

File: tomviz/RotateAlignWidget.cxx [hunk 1]
```diff
@@ -241,8 +241,35 @@ class RotateAlignWidget::RAWInternal
       this->reconSliceMapper[i]->SetSliceNumber(0);
       this->reconSliceMapper[i]->Update();
 
-      double range[2];
-      reconArray->GetRange(range);
+      double range[2] = {DBL_MAX, -DBL_MAX};
+
+      // Get the range of the inscribed circle only
+      auto radius = static_cast<double>(Nray) / 2;
```
Comment (cjh1) [inline tomviz/RotateAlignWidget.cxx:247]: For future reference, if you divide by `2.0` you don't need the `static_cast`

**Review by cjh1 — APPROVED** [review]
LGTM

**Pull request merged**